test/e2e/scenarios/AddRemoveReplica/cmd_decrease_replica.yaml

```diff
@@ -0,0 +1,44 @@
+id: "CMD_DECREASE_REPLICA"
+description: "Decrease replica count for CMD"
+resource:
+  apiVersion: $CRD_GROUP/$CRD_VERSION
+  kind: ReplicationGroup
+  metadata:
+    name: test$RANDOM_SUFFIX
```

Comment:
If the name matches the scenario, it helps in debugging using cli later in case of failure.
Please consider names such as: `cmd-dec-rep-$RANDOM_SUFFIX` for this scenario